src/components/LeafletMap.js

```diff
@@ -247,4 +249,20 @@ export class LeafletMap {
       dispatch.call("hide-loading-indicator");
     }
   };
+  findAutocompletLocation(resource) {
+    let center = resource.point.coordinates;
+    const markerIcon = L.icon({ iconUrl: "assets/empty-icon.svg" });
+    const marker = new L.marker(center, { icon: markerIcon }).addTo(this.map);
+    let markerContent = `
+        <div class="popup-container locality-popup-container">
+            <p class="popup-title"><strong>${resource.name}</strong></p>
+        </div>
+    `;
+    let bbox = resource.bbox;
```

Comment:
Just a suggestion but you could use ES6 de-structuring here:

```js
let [latSouth, longWest, latNorth, longEast] = resource.bbox;
this.map.fitBounds([
  [latSouth, longWest],
  [latNorth, longEast]
])
```

IMO the variable names help reduce cognitive load when reasoning with the code, but again it's just a suggestion.

Also, the order of the coordinates I used when passed to Leaflet's `map.fitBounds()` method are a bit different then how you've passed them. According to the [Bing docs for Location and Area Types](https://docs.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/location-and-area-types) the order of `bbox` coords are "South Latitude, West Longitude, North Latitude, East Longitude". And a coordinate pair in Leaflet is specified in the order of `latitude, longitude` (annoying, I know) but it seems like you're specifying them as `longitude, latitude` here. 

The search feature in the Netlify preview currently isn't working and I haven't checked out the branch locally so can't confirm if this works when coords are passed as `longitude, latitude`.

Reply:
don't use fitBounds anymore so not necessary